Automattic/jetpack
# Dashboard: Add a specific message for a site that is too new to have a rewind status

Fixes 6844-gh-Automattic/jpop-issues

#### Changes proposed in this Pull Request:

* Handle new `site_new` rewind status reason ( introduced in D66042-code ) on Jetpack Dashboard. In cases where a new user is sent immediately to the dashboard after being connected to a premium plan a timing issue could cause a message about installing VaultPress. Now it should let them know we are setting up things behind the scenes.

<img width="1036" alt="Screen Shot 2021-08-27 at 3 46 24 PM" src="https://user-images.githubusercontent.com/2810519/131195736-e5cddc48-0122-46a8-8875-9786b3bbb9af.png">

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






1. Create a new jurassic.ninja site with a premium plan and this branch
2. Open the Jetpack wp-admin dashboard in two tabs, one with a dev console open.
3. On the non-dev console tab connect Jetpack.
4. Immediately reload the other tab after clicking "Authorize"
5. Verify that the `rewind` network response returned `unavailable` and `new_site` reason.
6. Verify the Backup dash status says "Your site is new and may still be preparing its backup config."
